dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/FeatureManager.java

```diff
@@ -1535,7 +1541,9 @@ protected boolean updateFeatures(WsLocationAdmin locService,
                         bundleCache.addAllNoReplace(newBundleList);
 
                         // Update installedFeatures with the features that were successfully added
-                        featureRepository.setResolvedFeatures(goodFeatures, newConfiguredFeatures, reportedConfigurationErrors);
+                        featureRepository.setResolvedFeatures(goodFeatures,
```

Comment:
beta wrapped?

Reply:
Hmm, we didn't keep a 'setResolvedFeatures' with just the old parameters.  When beta is not enabled, the platforms parameter should be empty or null.  'setResolvedFeatures' does have a beta guard around the new code which records the platform and platform env var.